lib/Ohmbrewer_Sprouts.cpp

```diff
@@ -493,6 +504,17 @@ int Ohmbrewer::Sprouts::parseRIMSPins(char* params, std::list<int> &thermPins, i
     }
 
     errorCode = parsePumpPins(params, pumpPin);
+    if(errorCode != 0) {
+        return errorCode;
+    }
+
+    String safe = String(strtok(NULL, ","));
+    if (safe == NULL){
+        return -1;
+    }//TODO add other error codes?
+    safetyIndex = safe.toInt();
```

Comment:
If we don't provide the sensor bus pin in the argument string, this part can simply use the parseTemperatureSensorPins function.